packages/trader/src/App/Containers/Modals/__tests__/trade-modals.spec.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../trader-providers';
+import TradeModals from '../trade-modals';
+
+jest.mock('App/Components/Elements/Modals/UnsupportedContractModal', () =>
+    jest.fn(props => (
+        <div>
+            <div>Unsupported contract modal</div>
+            <button onClick={props.onConfirm}>onConfirm unsupported</button>
+            <button onClick={props.onClose}>onClose unsupported</button>
+        </div>
+    ))
+);
+jest.mock('App/Components/Elements/Modals/MarketUnavailableModal', () =>
+    jest.fn(props => (
+        <div>
+            <div>Market unavailable modal</div>
+            <button onClick={props.onConfirm}>onConfirm market</button>
+            <button onClick={props.onCancel}>onCansel market</button>
```

Comment:
just one typo, `onCancel market` in 3 places in the code

Reply:
Thank you, noted

Reply:
Thank you, noted)